### ISSUE TITLE
Migrating: Laika

### ISSUE THREAD

**Issue by rossabaker** [typelevel/governance#108]
## Migration Checklist

Project Submission Issue: #103 

### For Organization Projects:

- [x] Create a team for the repo with maintainer as team member
- [x] After repo transfer, add repo with write permissions to the created team

### For All Projects:

- [x] Add project to the site by [adding an entry to the projects.yml file](https://github.com/typelevel/typelevel.github.com/blob/development/_data/projects.yml)

### Announcements

These steps are optional but encouraged.

- [ ] Write a blog post announcing the change
- [x] Post to our social channels (Mastodon)

**Comment by rossabaker** [comment]
Since this is an existing project, a transfer makes more sense than a new project to keep the stars and history, right?  I think our template is a bit off here.

**Comment by rossabaker** [comment]
Never mind, my reading comprehension is poor today.

@jenshalm, I created a team and invited you so there's a way to tag you and any other maintainers.  Once the repo is transferred, I'll add it to the team.

**Comment by armanbilge** [comment]
I've requested to join the team as well :) congrats Jens!

**Comment by armanbilge** [comment]
> I assume I need to associate the mantainers/laika team after the transfer?

Yes, Ross (or another admin) will do this after the transfer:

> Once the repo is transferred, I'll add it to the team.

https://github.com/typelevel/governance/issues/108#issuecomment-1553725725

**Comment by jenshalm** [comment]
I have created a [PR for the third step](https://github.com/typelevel/typelevel.github.com/pull/449).

Apart from that I noticed that while all URLs starting with `github.com/planet42/Laika` are redirected the manual under `planet42.github.io/Laika` is not. I'll be looking into creating manual redirects in an organization site that contains a Laika folder. Otherwise all existing links to manual pages are just 404s.
